Add Cloud IDE note (#3679)
astronomer/docs: astro/cloud-ide/configure-project-environment.md

Configure your Astro Cloud IDE project environment, including Airflow dependencies, variables, and connections, in the Astro UI. You can use your environment configurations in any pipeline within your project.

Environment configuration in the Astro Cloud IDE is similar to Astro project and Airflow configuration, but more options are provided to minimize the time you spend configuring text files.

## Limitations

- You can't set OS-level dependencies with the Astro Cloud IDE.
- Only a subset of Airflow connections can be configured through Astro Cloud IDE templates. All other connections must be configured through a generic connection template.
- Environment configurations are not saved when you export a data pipeline as a DAG. To run your data pipeline outside of the Astro Cloud IDE, you must reconfigure these values in the Airflow environment where you exported your data pipeline.

> - To connect the Cloud IDE to a 3rd party resource, such as GitLab, the resource must be publicly accessible.

Reviewer: ```suggestion
```

> - In order to connect the Cloud IDE to a 3rd party resource, such as GitLab, the resource cannot be private. It must be public.

Reviewer: ```suggestion
- To connect the Cloud IDE to a 3rd party resource, such as GitLab, the resource must be publicly accessible.
```
